### PR TITLE
Improvements to deployment script

### PR DESCRIPTION
**Pull request by joesepi**

### CODE REVIEW

File: deploy-exclude-list.txt [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-composer.json
+gulpfile.js
+htaccess-local
+node_modules/
+package.json
 README.md
-
+vendor/
+wp-config.php
+wp-content/uploads/
```
Comment (joesepi) [inline deploy-exclude-list.txt:20]: I ordered these alphabetically.

File: deploy.sh [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+echo "Please enter username"
+read usr
+echo "Please enter server.name:diretory"
+read srvdir
+rsync -e ssh -p -avz --exclude-from deploy-exclude-list.txt ./_site/ $usr@$srvdir
+
+
+# Do it all with ssh keys:
+# https://www.digitalocean.com/community/tutorials/how-to-copy-files-with-rsync-over-ssh
+
```
Comment (joesepi) [inline deploy.sh:31]: Choose your adventure!

**Review by joesepi — COMMENTED** [review]
Just adding some context

File: deploy-exclude-list.txt (hunk 1 above)
Comment (jordalgo) [reply deploy-exclude-list.txt:20]: niiiiiice

File: deploy.sh [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+# Hardcode your server.name:directory (ex. mysite.com:public-www)
+# echo "Please enter username: "
+# read usr
+# rsync -e ssh -p -avz --exclude-from deploy-exclude-list.txt ./_site/ $usr@SERVER.NAME:DIRECTORY
+
+
+# Prompt for each detail
+echo "Please enter username"
+read usr
+echo "Please enter server.name:diretory"
```
Comment (jordalgo) [inline deploy.sh]: Will the server name and directory vary per repo?

Comment (joesepi) [reply deploy.sh]: No, but some people are overly cautious with their security, probably more so the directory than the server name. If you think it is overkill, I can remove it. Also, I need to spell directory correctly there! ;)

**Comment by jordalgo** [comment]
@joesepi Nice work.

**Pull request merged**